src/Compilers/Core/Portable/Emit/NoPia/EmbeddedTypesManager.cs

```diff
@@ -48,7 +48,7 @@ internal abstract partial class EmbeddedTypesManager<
         where TModuleCompilationState : CommonModuleCompilationState
         where TEmbeddedTypesManager : EmbeddedTypesManager<TPEModuleBuilder, TModuleCompilationState, TEmbeddedTypesManager, TSyntaxNode, TAttributeData, TSymbol, TAssemblySymbol, TNamedTypeSymbol, TFieldSymbol, TMethodSymbol, TEventSymbol, TPropertySymbol, TParameterSymbol, TTypeParameterSymbol, TEmbeddedType, TEmbeddedField, TEmbeddedMethod, TEmbeddedEvent, TEmbeddedProperty, TEmbeddedParameter, TEmbeddedTypeParameter>
         where TSyntaxNode : SyntaxNode
-        where TAttributeData : AttributeData, Cci.ICustomAttribute
+        where TAttributeData : class, Cci.ICustomAttribute
```

Comment:
>class [](http://example.com/codeflow?start=31&length=5)

Was there a reason for this change?

Reply:
> Was there a reason for this change?

Yes, I want to prevent the common component to take advantage of the APIs available from AttributeData.

Reply:
Among other things, this ensures that I adjusted all "interesting" code without creating error tests for all affected attributes.